PDSA_System/Server/Controllers/ForslagController.cs

```diff
@@ -76,6 +76,8 @@ public async Task<ActionResult<List<Forslag>>> GetBrukerForslag(int forfatterId)
     public async Task<ActionResult<List<Forslag>>> CreateForslag(Forslag forslag)
```

Comment:
Dersom bruker ikke er autentisert, får den en 500 som forårsakes en nullverdi i `forfatterId`
```suggestion
    [Authorize]
    public async Task<ActionResult<List<Forslag>>> CreateForslag(Forslag forslag)
```